pathplannerlib-python/pathplannerlib/util.py

```diff
@@ -154,6 +157,31 @@ def flipFeedforwards(feedforwards: List[float]) -> List[float]:
                 return [feedforwards[1], feedforwards[0]]
         return feedforwards
 
+    @classmethod
```

Comment:
Out of curiosity, why `@classmethod` here instead of `@staticmethod`?

Reply:
I did that to use `clazz.flipFeedforwards()` in the body, but you can replace it with `@staticmethod` and `FlippingUtil.flipFeedforwards()` instead if that's what you standardize on.